scripts/background.js

```diff
@@ -286,86 +342,36 @@ chrome.webRequest.onCompleted.addListener(function(details) {
                     //If show-all Context is true, Create a window which is focussed and create tabs in it
                     if(event2.showall==true){
                       chrome.windows.create({url:chrome.runtime.getURL("doi.html")+"?url="+message.url, width:800, height:800, top:0, left:0, focused:true},function(win){
-                        windowIdtest=win.id;
-                        chrome.windows.onRemoved.addListener(function (win1) {
-                          if(win1==windowIdtest){
-                            windowIdtest=0;
-                          }
-                        });
+                        windowIdtest = win.id;
                       });
                       var alexa_url="https://archive.org/services/context/alexa?url="+url;
```

Comment:
Could declare `alexa_url` globally and used in every where

Reply:
I will add a separate PR for this. At this time, It is all good.
Thank you @vbanos .